item-api/src/main/java/com/example/itemapi/global/config/LocalCacheConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.example.itemapi.global.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCache;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@EnableCaching
+public class LocalCacheConfig {
+
+    @Bean
+    public CacheManager cacheManager() {
+        SimpleCacheManager cacheManager = new SimpleCacheManager();
+        List<CaffeineCache> caches = Arrays.stream(CacheType.values())
+                .map(it -> new CaffeineCache(it.getCacheName(), Caffeine.newBuilder().recordStats()
```

Comment:
```suggestion
                .map(it -> new CaffeineCache(it.getCacheName(), Caffeine.newBuilder().recordStats()
```

Caffeine 를 사용해본적이 없어서 찾아봤는데요
다양한 기능을 제공해주네요

나중에 어떤 장단점이 있는지도 검토해보면 좋을거같아요

예 ) `recordStats()`  통계 함수의 부하 발생 같은...?